src/components/MetamaskLogin/template.pug

```diff
@@ -20,17 +20,22 @@ mixin metamask-missing
         )
 
 mixin wrong-chain
-    .icon
-        img(loading="lazy" src="{metamaskImage}" alt="metamask icon")
+    .icon.popup__icon
+        Icon(icon="{exclamation}" width="{60}" height="{60}" colorInherit)
     Heading(
-        class="title text-error"
+        class="title"
         level="{3}"
     ) Wrong network
     p.description 
-        | Please change your network to the {protocol} {' '}
+        | You are using a wrong network. Please connect to 
         b {networkName} {' '}
-        | network and refresh the page.
+        | to continue.
     .btns
+        Button(
+            label="How to?"
+            onClick="{init}"
```

Comment:
@tsejerome Please add a property for this onClick handler, this button should be a link

Reply:
What link shd it be, awaiting @markholdex or you for recommendation

Reply:
@tsejerome  for now please put this URL
`https://metamask.zendesk.com/hc/en-us/articles/360043227612-How-to-add-a-custom-network-RPC`